proofs/rustverifier.go

```diff
@@ -126,10 +127,7 @@ func (rp *RustVerifier) VerifyPoST(req VerifyPoSTRequest) (VerifyPoSTResponse, e
 	}
 
 	return VerifyPoSTResponse{
-		// TODO: change this to the bool statement
-		// See https://github.com/filecoin-project/go-filecoin/issues/1302
-		// bool(resPtr.is_valid),
-		IsValid: true,
+		IsValid: bool(resPtr.is_valid),
```

Comment:
does this always return true or are we really validating the `PoSTResponse` now?